packages/cap-config-writer/src/cap-writer/package-json.ts

```diff
@@ -53,12 +52,12 @@ async function updateScripts(
     packageJsonPath: string,
     projectName: string,
     appId: string,
-    enableNPMWorkspaces?: boolean
+    enableNPMWorkspaces?: boolean,
+    isCdsUi5PluginEnabled?: boolean
 ): Promise<void> {
-    const hasNPMworkspaces = await checkCdsUi5PluginEnabled(dirname(packageJsonPath), fs);
```

Comment:
@kjose90 I'm a bit confused by this code, the const naming and why did we do another call to `checkCdsUi5PluginEnabled` when this info is available, am I missing something?